contracts/number/types/Fixed18.sol

```diff
@@ -72,6 +72,16 @@ library Fixed18Lib {
         return Fixed18.wrap(Fixed6.unwrap(a) * 1e12);
     }
 
+    /**
+     * @notice Creates a signed fixed-decimal from a significand and an exponent
+     * @param significand Significand
+     * @param exponent Exponent
+     * @return New signed fixed-decimal
+     */
+    function from(Fixed18 significand, uint256 exponent) internal pure returns (Fixed18) {
```

Comment:
In all cases the exponent case be negative, see reference here:

https://github.com/equilibria-xyz/perennial-v2/blob/main/packages/perennial-oracle/contracts/pyth/PythFactory.sol#L74.